owncloudApp/build.gradle

```diff
@@ -115,12 +102,6 @@ android {
 
         multiDexEnabled true
 
-        javaCompileOptions {
-            annotationProcessorOptions {
-                arguments = ["room.schemaLocation": "$projectDir/schemas".toString()]
```

Comment:
I'm not sure if this is smart. 
Room needs it for migration tests and without it, you will probably see a warning during build concerning this

Reply:
Yes, i know room needs it for migration tests. That's why it is included in the ownCloudData module build gradle. https://github.com/owncloud/android/blob/master/owncloudData/build.gradle#L25